poc/pkg/values.yaml

```diff
@@ -80,3 +80,22 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+# Use for Istio service mesh 
+istio:
+  name: aks-istio-ingressgateway-external
+  namespace: aro-rp-poc
+  gateway:
+    name: rp-poc-gateway
+    #TODO jonachang: need to specify the host once we have the domain name
+    host: "*"
+    #TODO jonachang: change back to 443 when we have TLS
+    port: 80
+  virtualService:
+    name: rp-poc-virtualservice
+    port: 80
+    #TODO jonachang: need to update to the sample app name once created
+    service: aro-rp-poc-pkg
+  credential:
```

Comment:
Helm lint (https://helm.sh/docs/helm/helm_lint/) through an error here. I would just (1) remove the `credential` part and (2) add your comment to line 84/85.